boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm7.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mimxrt1180_evk/mimxrt1189/cm7
+name: NXP MIMXRT1180-EVK CM7
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 256
+flash: 256
```

Comment:
only 256? is that right?

Reply:
Yes, it's right. Because the cm7 core image is a ram target image like RT1170_CM4, I checked the flash size described in cm4 yml is 128.